package.json

```diff
@@ -19,6 +19,7 @@
         "blocked-at": "^1.2.0",
         "body-parser": "^1.19.0",
         "bootstrap": "^4.4.1",
+        "bootstrap4-toggle": "^3.6.1",
```

Comment:
I'd rather not include novel UI elements if there is a standard HTML element we can use instead. For example, I think we can just use a regular checkbox or radio in this case?